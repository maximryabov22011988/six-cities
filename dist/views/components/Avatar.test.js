"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const React = require("react");
const renderer = require("react-test-renderer");
const Avatar_1 = require("./Avatar");
it('Avatar renders correctly with mandatory props', () => {
    const tree = renderer
        .create(React.createElement(Avatar_1.default, { alt: "User avatar", name: "info@test.ru", parentClassName: "header", src: "/img" })).toJSON();
    expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Avatar.test.js.map