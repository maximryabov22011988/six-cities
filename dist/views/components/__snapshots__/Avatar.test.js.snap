// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly with mandatory props 1`] = `
Array [
  <div
    className="header__avatar-wrapper user__avatar-wrapper false"
  >
    <img
      alt="User avatar"
      className="reviews__avatar user__avatar"
      src="/img"
    />
  </div>,
  <span
    className="header__user-name"
  >
    info@test.ru
  </span>,
]
`;
